apps/condo/domains/organization/utils/testSchema/index.js

```diff
@@ -24,21 +25,35 @@ const OrganizationLink = generateGQLTestUtils(OrganizationLinkGQL)
 const TokenSet = generateGQLTestUtils(TokenSetGQL)
 /* AUTOGENERATE MARKER <CONST> */
 
-async function createTestOrganization (client, extraAttrs = {}) {
+async function readOrganization (client, id) {
+    if (!id) throw new Error ('no id')
+
+    return await Organization.getAll(client, { id })
+}
+
+/**
+ * Creates test organization prefilled with faker data, hydrated with extraAttrs and extraMeta, if provided
+ * @param client
+ * @param extraAttrs
+ * @param extraMeta {inn?: string, country?: string}
+ * @returns {Promise<({data: *, errors: *}|*|{country: (*|string), dv: number, sender: {dv: number, fingerprint: *}, meta: {zipCode: *, number: *, country: (*|string), dv: number, city, street: *, inn: (*), kpp: *}, name: *, description: *})[]>}
+ */
+async function createTestOrganization (client, extraAttrs = {}, extraMeta = {}) {
```

Comment:
No it’s a bad idea to change the common signature here. If you need to change meta, you need to use the second argument

Reply:
It’s a common style. Don’t change it.